cmd/frcli/utils.go

```diff
@@ -128,3 +143,42 @@ func clientAddressDialer(defaultPort string) func(context.Context,
 		)
 	}
 }
+
+// readMacaroon tries to read the macaroon file at the specified path and create
+// gRPC dial options from it.
+func readMacaroon(macPath string) grpc.DialOption {
```

Comment:
Since we're already importing `lnd/macaroons`, can we add a TODO to extract+export this logic in lnd? Just for the sake of less copied code 🧹

Reply:
Yes, I agree. I didn't want to introduce more dependencies so I copied it for now. But definitely something to keep in mind for a later cleanup. Added a TODO.